<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Client TCP UR5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/new/icones/res/uricon.png</normaloff>:/new/icones/res/uricon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxConnexion">
      <property name="title">
       <string>Connexion au bras UR5</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelStatut">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Statut connexion : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelStatutActuel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;em&gt;Non connecté&lt;/em&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonDeconnexion">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Déconnexion</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonConnexion">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Connexion</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Editeur URScript</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxProgramme">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Editeur de script</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QLabel" name="labelProgramFileName">
               <property name="text">
                <string>Aucun fichier URScript ouvert.</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QTextEdit" name="textEditProgramme">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButtonOuvrir">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Ouvrir un fichier URScript... (ALT+O)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonEnvoyerScript">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Envoyer URScript (ALT+E)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonPositionDepart">
               <property name="text">
                <string>Position de départ du robot (CTRL+D)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButtonArreterProgramme">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Arrêter l'URScript (ALT+S)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edition de texte</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxFontXML">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Police de caractères</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labelXMLFileName">
               <property name="text">
                <string>Aucun fichier XML ouvert</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelFontName">
               <property name="text">
                <string>Aucune police de caractère active</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonOuvrirXML">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Ouvrir un fichier XML... (ALT+X)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBoxChar">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBoxTextEditor">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Éditeur de texte</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEditText">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButtonGenerate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Générer URScript (ALT+G)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButtonPreview">
             <property name="minimumSize">
              <size>
               <width>567</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Prévisualisation courbes de Bézier (ALT+V)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="labelText">
             <property name="text">
              <string>Veuillez entrer le message à écrire :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBoxConsole">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditLogs">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionConnexion"/>
    <addaction name="actionDeconnexion"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuProgramme">
    <property name="title">
     <string>URScript</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="separator"/>
    <addaction name="actionEnvoyerScript"/>
    <addaction name="actionArreter_Programme"/>
    <addaction name="actionPositionDepart"/>
   </widget>
   <widget class="QMenu" name="menuPolice">
    <property name="title">
     <string>Édition de texte</string>
    </property>
    <addaction name="actionOuvrirXML"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionGenererScript"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuPolice"/>
   <addaction name="menuProgramme"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnexion"/>
   <addaction name="actionDeconnexion"/>
  </widget>
  <action name="actionConnexion">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/new/icones/res/connection.png</normaloff>:/new/icones/res/connection.png</iconset>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionDeconnexion">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/new/icones/res/logout.png</normaloff>:/new/icones/res/logout.png</iconset>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir un fichier URScript...</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionEnvoyerScript">
   <property name="text">
    <string>Envoyer URScript</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionArreter_Programme">
   <property name="text">
    <string>Arrêter l'URScript</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionOuvrirXML">
   <property name="text">
    <string>Ouvrir un fichier XML...</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionGenererScript">
   <property name="text">
    <string>Générer le script</string>
   </property>
   <property name="toolTip">
    <string>Générer le script à envoyer au robot</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Prévisualiser</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionPositionDepart">
   <property name="text">
    <string>Position de départ du robot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonConnexion</tabstop>
  <tabstop>pushButtonDeconnexion</tabstop>
  <tabstop>textEditProgramme</tabstop>
  <tabstop>pushButtonOuvrir</tabstop>
  <tabstop>pushButtonEnvoyerScript</tabstop>
  <tabstop>pushButtonPositionDepart</tabstop>
  <tabstop>pushButtonArreterProgramme</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButtonOuvrirXML</tabstop>
  <tabstop>comboBoxChar</tabstop>
  <tabstop>lineEditText</tabstop>
  <tabstop>pushButtonPreview</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>pushButtonQuitter</tabstop>
  <tabstop>textEditLogs</tabstop>
 </tabstops>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonConnexion</sender>
   <signal>clicked()</signal>
   <receiver>actionConnexion</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeconnexion</sender>
   <signal>clicked()</signal>
   <receiver>actionDeconnexion</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>actionQuitter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEnvoyerScript</sender>
   <signal>clicked()</signal>
   <receiver>actionEnvoyerScript</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOuvrir</sender>
   <signal>clicked()</signal>
   <receiver>actionOuvrir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonArreterProgramme</sender>
   <signal>clicked()</signal>
   <receiver>actionArreter_Programme</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPositionDepart</sender>
   <signal>clicked()</signal>
   <receiver>actionPositionDepart</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOuvrirXML</sender>
   <signal>clicked()</signal>
   <receiver>actionOuvrirXML</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPreview</sender>
   <signal>clicked()</signal>
   <receiver>actionPreview</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGenerate</sender>
   <signal>clicked()</signal>
   <receiver>actionGenererScript</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
